common/src/main/java/com/wynntils/commands/PlayerCommand.java

```diff
@@ -154,18 +149,14 @@ private int lookupPlayerLastSeen(CommandContext<CommandSourceStack> context) {
                             .withStyle(ChatFormatting.GRAY)
                             .append(Component.literal(player.server()).withStyle(ChatFormatting.GOLD)));
                 } else {
-                    try {
-                        Date joinedDate = DATE_FORMAT.parse(player.lastJoinTimestamp());
-                        long differenceInMillis =
-                                System.currentTimeMillis() - joinedDate.getTime() + getTimezoneOffset();
-
-                        response.append(Component.literal(" was last seen ").withStyle(ChatFormatting.GRAY))
-                                .append(Component.literal(DATE_FORMATTER.format(differenceInMillis))
-                                        .withStyle(ChatFormatting.GOLD)
-                                        .append(Component.literal("ago").withStyle(ChatFormatting.GRAY)));
-                    } catch (ParseException e) {
-                        WynntilsMod.error("Error when trying to parse player last join", e);
-                    }
+                    long differenceInMillis = System.currentTimeMillis()
```

Comment:
I'm pretty certain that there is a better way to get the elapsed time from a proper java.time timestamp and the current system time, than doing manual calculations and correcting for time zone etc. I don't recall the API details, but have a look at e.g. `Duration`.